src/authentication.js

```diff
@@ -291,28 +291,38 @@ export class Authentication {
   /**
    * Redirect (page reload if applicable for the browsers save password option)
    *
-   * @param {[string]} redirectUrl The redirect url
-   * @param {[string]} defaultRedirectUrl The defaultRedirectUrl
+   * @param {string}   redirectUrl The redirect url. To not redirect use an empty string.
+   * @param {[string]} defaultRedirectUrl The defaultRedirectUrl. Used when redirectUrl is undefined
    * @param {[string]} query The optional query string to add the the url
    * @returns {undefined} undefined
    *
    * @memberOf Authentication
    */
-  redirect(redirectUrl?: string, defaultRedirectUrl?: string, query?: string) {
+  redirect(redirectUrl: string, defaultRedirectUrl?: string, query?: string) {
```

Comment:
@doktordirk This is a BC.. An optional parameter is now mandatory, so TS will start complaining if someone has called it without parameters.